### PR TITLE
Appraisals: add rails-5 appraisal

### PR DESCRIPTION
**Pull request by kyrylo**
Rails 5 was released a few days ago.
`rails-edge` is `rails-5.1.0.alpha` for the time being.

### CODE REVIEW

File: Appraisals [hunk 1]
```diff
@@ -66,17 +66,33 @@ appraise 'rails-4.2' do
...
+  appraise 'rails-5.0' do
+    gem 'rails', '~> 5.0.0'
+    gem 'warden', '~> 1.2.6'
+    gem 'rack', '~> 2.0'
+
+    gem 'activerecord-jdbcsqlite3-adapter', '~> 1.3.20', platforms: :jruby
+    gem 'sqlite3', '~> 1.3.11', platforms: [:mri, :rbx]
+
+    gem 'resque', '~> 1.26'
+    gem 'resque_spec', git: 'git@github.com:kyrylo/resque_spec.git'
```
Comment (shifi) [inline Appraisals]: Maybe I missed something, but why is this not in airbrake org?

File: Appraisals [hunk 2]
```diff
@@ -66,17 +66,33 @@ appraise 'rails-4.2' do
 end
 
 # Rails 5+ supports only Ruby 2.1+
```
Comment (shifi) [inline Appraisals]: I think this should be [`Ruby 2.2.2+`](https://github.com/rails/rails/commit/32f7491808d2c4e097ed7b3ee875e4d1cea8c442#diff-73ac7236f7a8247a9cd61d338ef49cda).

File: Appraisals [hunk 3]
```diff
@@ -66,17 +66,33 @@ appraise 'rails-4.2' do
...
-if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.0')
+if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.3.0')
```
Comment (shifi) [inline Appraisals]: `2.2.2`?

File: Appraisals (hunk 1 above)
Comment (kyrylo) [inline Appraisals]: Changed to `airbrake/resque_spec`.

File: Appraisals (hunk 2 above)
Comment (kyrylo) [inline Appraisals]: Fixed.

File: Appraisals (hunk 3 above)
Comment (kyrylo) [inline Appraisals]: Fixed.

**Comment by shifi** [comment]
LGTM

**Pull request merged**